frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -220,32 +128,28 @@ const handleSubmit = async () => {
         errorForestClientIdList: [] as string[],
     };
     do {
-        const item = forestClientData.value.pop();
-        const itemForestClientNumber = item?.forest_client_number
-            ? item.forest_client_number
-            : '';
-        const data = toRequestPayload(formData.value, item);
-
+        const forestClientNumber = formData.value.verifiedForestClients.pop();
```

Comment:
Hi @MCatherine1994 , I probably never try this before, but would this also works if we really need an empty string for the const, so we don't need to repeat it?
`formData.value.verifiedForestClients.pop() || ''`
Thanks.

Reply:
well, at this step it needs to be undefined so when we pass to the toRequestPayload method, it will be ignored. The logic here is a little bit confused. Later on we need some refactoring as I mentioned above. For this ticket will just follow what we have now.

Reply:
I was confused a lot yesterday, and got the error when use your suggestion. And then I realized there are some logic that we need to rework on.